ui/src/views/Pages/Login/Login.js

```diff
@@ -18,7 +18,7 @@ class Login extends Component {
   login = (e) => {
     console.log("Login")
     e.preventDefault()
-    this.setState({ loggedIn: this.state.username != '' })
+    this.setState({ loggedIn: this.state.username !== '' })
```

Comment:
There are few words to describe how much I dislike comparison of things in JavaScript.